docs/5.x/instance.md

```diff
@@ -4,7 +4,7 @@
 
 ### Create instance
 
-Choose debian image from community images:
+Choose a debian image from community images:
```

Comment:
This entire file probably shouldn't be in the OSS repo as it appears to be mostly internal, and looks to be largely out of date?

Reply:
It looks like it but I'd do this separately.